tests/tck/features/mutate/ClearSpace.feature

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2022 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License.
+Feature: Clear space test
+  This file is mainly used to test the function of clear space.
+
+  Scenario: Clear space syntax test
+    Given an empty graph
+    And create a space with following options:
+      | name           | clear_space      |
+      | partition_num  | 9                |
+      | replica_factor | 1                |
+      | vid_type       | FIXED_STRING(20) |
+    And wait 2 seconds
+    When executing query:
+      """
+      CLEAR SPACE IF EXISTS clear_space;
```

Comment:
Need you to consider the role of this operation? Could anyone clear the used space or only root user?

Reply:
only root can do this.  https://confluence.nebula-graph.io/pages/viewpage.action?pageId=40535032